src/pages/settings/Profile/Contacts/NewContactMethodPage.js

```diff
@@ -52,99 +54,79 @@ const defaultProps = {
     loginList: {},
 };
 
-class NewContactMethodPage extends Component {
-    constructor(props) {
-        super(props);
+function NewContactMethodPage(props) {
```

Comment:
Suuuper just wondering, why aren't we doing this? I see some components doing this & some doing what you have, I feel like it would be nice to standardize but maybe it doesn't matter for now

```suggestion
const NewContactMethodPage = (props) => {
```

Reply:
The difference between these 2 things is that defining it as a function means that it will be hoisted into the global scope. Defining it as a variable prevents this hoisting. See for the nitty gritty on why is may or may not matter https://github.com/getify/You-Dont-Know-JS/blob/2nd-ed/scope-closures/ch5.md#hoisting-declaration-vs-expression

But I agree, right now we are fine with either and we should probably standardize on one or the other. I don't have a strong preference for either format.